### ISSUE TITLE
Allow admins to manage other users’ preferences

### ISSUE THREAD

**Issue by brandonkelly** [craftcms/cms#6905]
So perhaps allowing these preferences to be controlled by 'admin' staff as well would be good. It would be ideal if these could be setup and added when I create a user account or group.

In my opinion the 'Use shapes to represent statuses' should be the default value anyway.

_Originally posted by @terryupton in https://github.com/craftcms/cms/issues/6896#issuecomment-698326708_
